test-harness/modules/generic_test_code.py

```diff
@@ -9,6 +9,17 @@
 
 
 def generic_unknown_user_is_forbidden_test(ar, auth_header, path):
+    """Test if invalid user is forbidden access
```

Comment:
user stems from uid claim in JWT token - it's a random string unknown to the system.